examples/bridge/animation-controls/simple.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Airship bridge</title>
+
+  <script src="https://cdn.jsdelivr.net/npm/lodash@4.17.11/lodash.min.js"></script>
+  <script src="https://libs.cartocdn.com/carto-vl/v1.3/carto-vl.min.js"></script>
+
+  <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v0.52.0/mapbox-gl.css" rel="stylesheet" />
+  <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v0.52.0/mapbox-gl.js"></script>
+
+  <link rel="stylesheet" href="../../../packages/styles/dist/airship.css">
+  <script type="module" src="../../../packages/components/dist/airship/airship.esm.js"></script>
+  <script nomodule="" src="../../../packages/components/dist/airship/airship.js"></script>
+  <script src="../../../packages/bridge/dist/asbridge.js"></script>
+
+  <style>
+    .as-map-footer {
+      max-width: 812px;
+    }
+  </style>
+</head>
+
+<body class="as-app-body">
+  <div class="as-app">
+    <div class="as-content">
+      <main class="as-main">
+        <div class="as-map-area">
+          <div id="map"></div>
+        </div>
+        <div class="as-map-footer">
+          <div class="as-box">
+            <as-animation-controls-widget
+              animated
+              heading='Animation'
+              description='Try seeking, pausing or selecting a range'
```

Comment:
Selecting a range doesn't work on this one so we should remove it

```suggestion
              description='Try seeking and pausing'
```